include/oboe/AudioStream.h

```diff
@@ -395,6 +395,16 @@ class AudioStream : public AudioStreamBase {
 
     AudioFormat mNativeFormat = AudioFormat::Invalid;
 
+    /**
+     * Update mFramesWritten.
```

Comment:
Does this need to be a public method? It feels like this should only ever be called by the stream class (or subclass) to update its own frame counter from the underlying stream, which isn't something a client should be concerned with. If you agree, suggest changing to `protected`.

Reply:
It is already "protected:".